sqlite-db/src/lib.rs

```diff
@@ -30,14 +30,11 @@ use std::path::PathBuf;
 use std::sync::Arc;
 use time::Duration;
 
-pub use closed::*;
-pub use failed::*;
+pub use db::closed::*;
+pub use db::failed::*;
 
-mod closed;
-mod event_log;
-mod failed;
-
-pub mod time_to_first_position;
+mod db;
```

Comment:
🔧 I would flatten the module structure, getting rid of `db`. 

Might be better to do it in a follow-up commit to make the diff easier to review, but not sure.

Reply:
I can do that 👍